### PR TITLE
Add conditions during PEL amendments

### PR DESCRIPTION
**Pull request by MaxOSullivan**
- Add conditions to PEL task pages
- Edit task router to include condition and reminder data

### CODE REVIEW

File: pages/task/read/views/models/establishment.jsx [hunk 1]
```diff
@@ -52,6 +56,21 @@ export default function Establishment({ task, values }) {
             proposedLabel={isComplete ? <Snippet>diff.changed-to</Snippet> : undefined}
           />
 
+          <div className="sticky-nav-anchor">
+            <h2><Snippet>conditions.title</Snippet></h2>
+            <Conditions
+              conditions={taskData.conditions ? taskData.conditions : establishment.conditions }
+              reminders={taskData.reminder !== '' ? [JSON.parse(taskData.reminder)] : establishment.reminders}
```
Comment (gaz1634) [inline pages/task/read/views/models/establishment.jsx:63]: Does this work if there are no reminders in the task?
Is the JSON.parse necessary?

File: pages/task/read/views/models/place.jsx [hunk 2]
```diff
@@ -97,6 +100,22 @@ export default function Playback({ task, values, allowSubmit }) {
           setDirty={setDirty}
         />
       </StickyNavAnchor>
+    ),
+    (
+      <StickyNavAnchor id="conditions" key="conditions">
+        <h2><Snippet>conditions.title</Snippet></h2>
+        <Conditions
+          conditions={taskData.conditions ? taskData.conditions : establishment.conditions }
+          reminders={taskData.conditions ? [JSON.parse(taskData.reminder)] : establishment.reminders}
```
Comment (gaz1634) [inline pages/task/read/views/models/place.jsx:109]: Similar to my previous comment. Is parse necessary?

Also should the logic on populating the Condition components reminders property be consistent?

File: pages/task/read/views/models/role.jsx [hunk 3]
```diff
@@ -96,6 +99,22 @@ export default function Role({ task, values, schema }) {
           }
         </StickyNavAnchor>
       )
+    ),
+    (
+      <StickyNavAnchor id="conditions" key="conditions">
+        <h2><Snippet>conditions.title</Snippet></h2>
+        <Conditions
+          conditions={taskData.conditions ? taskData.conditions : establishment.conditions }
+          reminders={taskData.conditions ? [JSON.parse(taskData.reminder)] : establishment.reminders}
```
Comment (gaz1634) [inline pages/task/read/views/models/role.jsx:108]: Again, should this logic be consistent with other reminders properties?

File: pages/task/read/views/models/establishment.jsx (hunk 1 above)
Comment (MaxOSullivan) [reply pages/task/read/views/models/establishment.jsx]: Have made the passing of reminders consistent
Parse is needed due to the way the data is passed through at the bottom of this file https://github.com/UKHomeOffice/asl-components/pull/277/files#diff-702fbff05fa84184acb68e5f199300abac056b3eabe558361ded0327cafce9f3

The object needs to be added as part of the form to be sent in the request when the amendment is accepted or returned
Happy to change it if there's an easier way

**Review by gaz1634 — APPROVED** [review]

**Pull request merged**